app/views/recipe_ingredients/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='container wrapper'>
+  <div class='row'>
+    <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
+      <div class='row'>
+        <div class='col-lg-12'>
+          <%= form_for @recipe_ingredient, url: {action: 'create'} do |rec_ing| %>
```

Comment:
I would still suggest to rename `rec_ing` to `form`. We already have `recipe_ingredient` so it collides a little bit.